### ISSUE TITLE
[Table]: Contents of the table are not announced by screen reader

### ISSUE THREAD

**Issue by lateefsofi** [SAP/ui5-webcomponents-react#6387]
### Describe the bug

Content of the table are not announced by screen reader. When user navigates from one row to another, only header text is announced by not the content

### Isolated Example

https://stackblitz.com/edit/github-xfs7ff?file=src%2FToDos.module.css,src%2Fmain.tsx,src%2FToDos.tsx

### Reproduction steps

1. Navigate to https://stackblitz.com/edit/github-xfs7ff?file=src%2FToDos.module.css,src%2Fmain.tsx,src%2FToDos.tsx
2. Start Voiceover on Mac or Jaws on windows
3. Navigate through the table and check the announcement

### Expected Behaviour

Row content should be announced with the column headers

### UI5 Web Components for React Version

1.29.8

### UI5 Web Components Version

1.24.4

### Browser

Chrome

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by lateefsofi** [comment]
Looks we have to warp the contents in span tag then only screen will announce it

**Comment by Lukas742** [comment]
Hi @lateefsofi 

could you please explain why you've reopened the issue, so we can investigate the problem?

**Comment by lateefsofi** [comment]
@Lukas742 
Interactive elements in the cells like Switc and Button in the current [example](https://stackblitz.com/edit/github-xfs7ff?file=src%2FToDos.module.css,src%2Fmain.tsx,src%2FToDos.tsx) are not announced.
User had to press tab key to focus over Switch button then only it will be announced. 
a11y tester shared the UI5 behaviour example https://ui5.sap.com/#/entity/sap.m.plugins.UploadSetwithTable/sample/sap.m.sample.UploadSetwithTablePlugin.ResponsiveTableSimple, we expect ui5 web-components table to make announcements like this.

**Comment by Lukas742** [comment]
@lateefsofi I've tested with VoiceOver and there it seems to announce the interactive content correctly. Please see the attached video and let me know if I've missed something:

https://github.com/user-attachments/assets/70781c2a-376b-475e-9ec0-64a2646e7dd1

**Comment by lateefsofi** [comment]
@Lukas742  at 0.25 if you see the announcement is only for first column cell but not for other 2 columns(only header is announced)

**Comment by Lukas742** [comment]
@lateefsofi thanks for clarifying.

Since the `Table` and all of its sub components are web components developed by the ui5-webcomponents team, I'm going to forward this issue to their repo.

___

Hi Colleagues,
here you can find the given example, just without React or our wrapper: https://stackblitz.com/edit/github-7aqd1q?file=main.js,index.html
